src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -67,48 +77,69 @@ export const MultiHopTradeConfirm = memo(() => {
     [tradeExecutionState],
   )
 
+  const handleTradeConfirm = useCallback(() => {
+    dispatch(tradeQuoteSlice.actions.confirmTrade())
+  }, [dispatch])
+
+  const handleSubmit = useCallback(() => {
+    if (isModeratePriceImpact) {
+      setShouldShowWarningAcknowledgement(true)
+    } else {
+      handleTradeConfirm()
+    }
+  }, [handleTradeConfirm, isModeratePriceImpact])
+
   return (
     <TradeSlideTransition>
-      <Card
-        flex={1}
-        borderRadius={cardBorderRadius}
-        width='full'
-        variant='dashboard'
-        maxWidth='500px'
+      <WarningAcknowledgement
+        message={`This trade is impacted by price movement (${bnOrZero(priceImpactPercentage)
+          .toFixed(2)
+          .toString()}%). Proceed with caution.`}
```

Comment:
also translation

Reply:
Actioned in https://github.com/shapeshift/web/pull/6647/commits/05c292f7035995dab341ddf89c2ad7e64e3e383b.